tests/integration/pilot/istioctl_test.go

```diff
@@ -84,7 +84,7 @@ spec:
         host: reviews
 `)
 			istioCtl := istioctl.NewOrFail(ctx, ctx, istioctl.Config{Cluster: ctx.Environment().Clusters()[0]})
-			istioCtl.InvokeOrFail(t, []string{"x", "wait", "VirtualService", "reviews." + ns.Name()})
+			istioCtl.InvokeOrFail(t, []string{"x", "wait", "-v", "VirtualService", "reviews." + ns.Name()})
```

Comment:
Do we also want a test for the new `--generation` flag?

Reply:
The generation flag is thoroughly covered by unit tests, and is not expected to be used much in practice.